src/redturtle/prenotazioni/content/booking_type.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+from plone.app.textfield import RichText
+from plone.dexterity.content import Item
+from plone.supermodel import model
+from zope import schema
+from zope.interface import implementer
+
+from redturtle.prenotazioni import _
+
+
+class IBookingType(model.Schema):
+    """Marker interface and Dexterity Python Schema for Prenotazione"""
+
+    duration = schema.Choice(
+        title=_("Duration value"),
+        required=True,
+        vocabulary="redturtle.prenotazioni.VocDurataIncontro",
+    )
+
+    requirements = RichText(
+        required=False,
+        title=_("Cosa serve", default="Cosa serve"),
```

Comment:
No mistone: o tutte in inglese o tutte in italiano di default, please.
Farei default in inglese.
E poi di conseguenza anche le traduzioni.

Reply:
Just copied from existing code :D

Reply:
Done